packages/nimble-components/src/rich-text-viewer/styles.ts

```diff
@@ -1,12 +1,60 @@
 import { css } from '@microsoft/fast-element';
 import { display } from '@microsoft/fast-foundation';
-import { bodyFont, bodyFontColor } from '../theme-provider/design-tokens';
+import {
+    bodyFont,
+    bodyFontColor,
+    linkActiveFontColor,
+    linkFontColor,
+    smallPadding
+} from '../theme-provider/design-tokens';
 
 export const styles = css`
     ${display('flex')}
 
     :host {
         font: ${bodyFont};
+        outline: none;
         color: ${bodyFontColor};
+        inline-size: auto;
+        overflow: auto;
+        block-size: 100%;
+        min-block-size: 36px;
+    }
+
+    .viewer {
+        font: inherit;
+        outline: none;
+        box-sizing: border-box;
+        position: relative;
+        color: inherit;
+        min-inline-size: 100px;
+        padding: ${smallPadding};
+        margin-block-end: 10px;
+        margin-inline-end: 10px;
+    }
+
+    .viewer > :first-child {
+        margin-block-start: 0;
+    }
+
+    .viewer > :last-child {
+        margin-block-end: 0;
+    }
+
+    li > p {
+        margin-block: 0;
+    }
+
+    li > p:empty {
+        display: contents;
+    }
+
+    a {
```

Comment:
Is the goal to eventually render actual `nimble-anchor` elements instead and what is shown here is temporary?

If not we need a plan to share styling more closely and prevent discrepancies.

Reply:
@rajsite Yes, this could be a temporary solution to render the links with similar styles as `nimble-anchor`. However, we are uncertain about the appropriate approach to proceed. Here are a couple of possible directions:

1. If we intend to render an actual `nimble-anchor` instead of an `a` tag, we can consider selecting all the `a` tags from the `serializedContent` and replacing them with newly created `nimble-anchor` elements. This replacement would involve copying the contents and attributes (especially href) from the converted `a` tags from the markdown string to the newly created ones.
2. If the goal is simply to have common shared styles, is it possible to place all `nimble-anchor` related styles in some shared folders which allow us to import in nimble-components wherever links are used? 

I added these styles to `a` tags because we just wanted to keep the implementation simple by styling the links using shared tokens without altering the document fragments through an additional replacement of `<a>` tags with `<nimble-anchor>`. However, we would like to hear your thoughts on the right plan to address the link behaviors in nimble components and also what must be included in this PR.

Reply:
> If we intend to render an actual nimble-anchor instead of an a tag, we can consider selecting all the a tags from the serializedContent and replacing them with newly created nimble-anchor elements.

This sounds like a hack / workaround. My concerns are how would you get consistency with the editor rendering of links? I would expect to be consistent in both the edit and view modes for rendering anchors with the same elements.

Are you suggesting that there are not ways to customize the HTML that is rendered by ProseMirror? Searching around a bit it seems like maybe we can customize the link mark: https://prosemirror.net/examples/schema/#:~:text=Links

> is it possible to place all nimble-anchor related styles in some shared folders

That may be possible. The [highlighted lines](https://github.com/ni/nimble/blob/86f2f56c93053c132f96fd906bab2c7e535f866c/packages/nimble-components/src/anchor/styles.ts#L27C1-L54C6) are the default link style behaviors. We can create a new anchor [pattern](https://github.com/ni/nimble/tree/main/packages/nimble-components/src/patterns)  and move those styles there to share. We should modify all those styles to target `a` instead of targeting `.control` (both in the shared anchor pattern and in the remaining anchor styles in the anchor component).

Sounds like shared styles might be a bit easier. It doesn't need to be in this PR to keep the changes smaller but the work needs to be captured somewhere and you should link to it from this comment.

Reply:
Yes, we can plan to implement the shared style approach. I have created a new issue to track this work: https://github.com/ni/nimble/issues/1372